e2e/regression_test/settings.no_basal_rates.spec.js

```diff
@@ -1,8 +1,10 @@
-const { setup, match, loopSettings, FilterSettings, SettingDefault, SettingType } = require('../../src/index');
+const { setup, loopSettings, Status, FilterSettings, SettingDefault, SettingType } = require('../../src/index');
 
 describe('Closed loop is not allowed when settings', () => {
+    var status;
     beforeAll(async () => {
         await setup.LaunchLoop();
+        status = new Status();
```

Comment:
[nit] Perhaps `loopStatus` to mirror `loopSettings`?  Otherwise `status` is not too specific.

Reply:
hmmm this is for functions relating to the "status" screen. Should i rename all to be `StatusScreen` `SettingsScreen` etc ?

Reply:
Is the status screen the home screen? The top part of the home screen? Not sure what?

Reply:
I thought so, in that case I can just rename it as `HomeScreen`